app/pb_kits/playbook/pb_person_contact/_person_contact.html.erb

```diff
@@ -1,25 +1,26 @@
 <%= content_tag(:div,
-    id: object.id,
-    data: object.data,
-    class: object.classname) do %>
-    <%= pb_rails("person", props: {
-        first_name: object.first_name,
-        last_name: object.last_name,
+  id: object.id,
+  data: object.data,
+  class: object.classname) do %>
+  <%= pb_rails("person", props: {
+      first_name: object.first_name,
+      last_name: object.last_name,
+  }) %>
+  <% object.valid_contacts.each do |contact| %>
+    <%= pb_rails("contact", props: {
+        contact_type: contact[:contact_type],
+        contact_value: contact[:contact_value],
+        contact_detail: contact[:contact_detail],
     }) %>
-    <% object.contacts.each do |contact| %>
+  <% end %>
+  <% unless object.wrong_contacts.empty? %>
+    <div style="padding-top: 10px"> <%= pb_rails("caption", props: { text: "wrong number" }) %></div>
```

Comment:
We never want to add inline styles. Please move this to the person_contact.scss https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_person_contact/_person_contact.scss

Also, use a spacing token for the padding value

Reply:
`$space-xs` looked better than `$space-md`, let me know what you think, I'll update the screen shot.

Reply:
@jasoncypret I think this is in its final form as is... I would like to get it in the next version raise in Nitro tomorrow. I'm going to head out in a few minutes here but I could try to make any last-minute changes tonight so it could be included tomorrow's version raise. Thanks!